admin/app/styles/authenticated/users/list.scss

```
@@ -1,11 +1,21 @@
+.pix-select {
+  min-width: 110px;
+  margin-right: 16px;
+}
+
+.pix-select .pix-icon {
+  width: 1.35rem;
+  height: 1.35rem;
+}
+
```

Ne faudrait-il pas autant que possible utiliser des tokens du Design System ?
cf. https://ui.pix.fr/?path=/docs/design-tokens-espacements--docs

```suggestion
.pix-select {
  min-width: 110px;
  margin-right: var(--pix-spacing-4x);
}

.pix-select .pix-icon {
  width: var(--pix-spacing-6x);
  height: var(--pix-spacing-6x);
}

```

Pour les grandes valeurs comme `110px` et `150px` je ne sais pas comment faire :confused:

---

D'accord avec toi pour utiliser généralement les tokens.
Mais le width et la height ne font généralement pas l'objet d'un token vu qu'ils peuvent dépendre du format / type d'écran mobile / tablette / maquette...

Dans ce cas c'est du custom, généralement en `rem` pour des raisons d'a11y, afin que le container s'ajuste à la taille par défaut de la police du navigateur.

Je conseille la lecture de cet article sur le sujet : https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/ 😄